### DISCUSSION TITLE
MacOS dictate feature

### DISCUSSION THREAD

**Discussion by madflow** [ghostty-org/ghostty#3728] [Feature Requests, Ideas] [answered]
I would like to use the "dictate" feature in MacOS with Ghostty (when writing docs in Neovim). First of all it is somewhat working for me, which is great.

But when I trigger the dictate function, there is no imediate feedback. When I start the dictate function with a keybinding I have to trigger it again. Only then I can see the text.
This works differently in Terminal.app for example. There I can immediatly see the spoken text.

Feature request: Have it behave similar like in Terminal.app with immediate feedback on the spoken text.

```
System Version: macOS 15.1.1 (24B91)
Kernel Version: Darwin 24.1.0
Model Name: MacBook Pro
Model Identifier: MacBookPro18,3                                                    
Chip: Apple M1 Pro   
```

**Comment by mitchellh** [comment, marked as answer]
I'm not sure when, but this has been resolved. Dictation works as it does in Terminal.app and other programs in nightly Ghostty.
